aea/configurations/base.py

```diff
@@ -1537,7 +1537,18 @@ def default_ledger(self, ledger_id: str):
 
     def component_configurations_json(self) -> List[OrderedDict]:
         """Get the component configurations in JSON format."""
-        return list(map(OrderedDict, self.component_configurations.values()))
+        result: List[OrderedDict] = []
```

Comment:
the difference with the previous implementation is that earlier we included `author`, `name` and `version` in the values of the dictionary `self.component_configurations`; however, to avoid inconsistencies, those information about the package can be extracted from the key.